app/src/main/java/org/wikipedia/dataclient/Service.kt

```diff
@@ -141,7 +141,7 @@ interface Service {
     fun getVideoInfo(
         @Query("titles") titles: String,
         @Query("viextmetadatalanguage") lang: String
-    ): Observable<MwQueryResponse>
+    ): MwQueryResponse
```

Comment:
This function should also be marked as `suspend`, otherwise it will not load videos correctly in the Gallery.

Reply:
Done!